.github/workflows/ci.yml

```diff
@@ -179,6 +179,21 @@ jobs:
       - name: Ccache stats
         run: ccache -s
 
+      - name: Create ccache debug file
+        if: runner.os == 'Linux' && matrix.shard == 1
+        # Create the preprocessed file for the primitive_events.cc file.
```

Comment:
This is just one arbitrarily chosen file, right? Maybe explain that in the comment.

Reply:
done.